circle.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
     - npm i --production
     - CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
     - nohup bash -c "CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt serverproc:dist &"
-    - npm install intern xmlhttprequest firefox-profile request sync-exec convict yargs leadfoot
+    - npm install intern xmlhttprequest firefox-profile request sync-exec convict yargs leadfoot otplib
```

Comment:
@vbudhram 
![image](https://user-images.githubusercontent.com/128755/37629244-1aaf57b0-2bb4-11e8-99b9-b3fdc2c45123.png)

could you add the otplib to `run.sh` and `install-npm-deps.sh`.